pages/badges/badge-single.jsx

```diff
@@ -0,0 +1,449 @@
+'use strict';
+
+var React = require('react'),
+    Link = require('react-router').Link,
+    LoginLink = require('../../components/login/LoginLink.jsx'),
+
+    RequirementsList = require('../../components/badges/requirement-list.jsx'),
+    SocialShare = require('../../components/badges/social-share.jsx'),
+    Badge = require('../../components/badges/badge.jsx'),
+    BadgeHorizontalIcon = require('../../components/badges/badge-horizontal-icon.jsx'),
+    CredlyLinkForm = require('../../components/badges/CredlyLinkForm.jsx'),
+
+    Divider = require('../../components/Divider.jsx'),
+    BadgesAPI = require('../../lib/badges-api'),
+    TeachAPI = require('../../lib/teach-api'),
+    Modal = require('../../components/modal.jsx');
+
+var Navigation = React.createClass({
+  render: function() {
+    var prev = this.props.prev;
+    var next = this.props.next;
+    return (
+      <div className="badge-navigation">
+        <a className="previous" href={prev.url}>
+          <img src={prev.img} />
+          <span clasName="label">← {prev.title}</span>
+        </a>
+        <a className="next" href={next.url}>
+          <span clasName="label">{next.title} →</span>
+          <img src={next.img} />
+        </a>
+      </div>
+    );
+  }
+});
+
+/**
+ * Single badge Page
+ * should take a badgeId
```

Comment:
actually an outdated comment after investigating - this component is passed a full badge object now